specification/api-tracing.md

```diff
@@ -386,6 +387,18 @@ Attributes SHOULD preserve the order in which they're set. Setting an attribute
 with the same key as an existing attribute SHOULD overwrite the existing
 attribute's value.
 
+Attribute values expressing a numerical value of zero or an empty string are
+considered meaningful and MUST be stored and passed on to span processors / exporters.
+Attribute values of `null` are considered to be not set and get discarded as if
+`SetAttribute` had never been called.
+If overwriting values is allowed, this results in clearing the previous value
+and dropping the attribute key from the set of attributes.
+
+`null` values within arrays MUST be preserved. This is required for map/dictionary
```

Comment:
preserved how? As empty strings/0 values or as-is? Please specify (or say that it's implementation-defined).